tests/release/pipelines/fbc_release.go

```diff
@@ -15,184 +15,270 @@ import (
+		Expect(err).ShouldNot(HaveOccurred())
+		snapshot, err := devFw.AsKubeDeveloper.IntegrationController.GetSnapshot("", buildPr.Name , "", devNamespace)
+		Expect(err).ShouldNot(HaveOccurred())
+		releaseCR, err := devFw.AsKubeDeveloper.ReleaseController.GetRelease("", snapshot.Name, devNamespace)
+		if err != nil {
+			return err
+		}
+		GinkgoWriter.Println("Release CR: ", releaseCR.Name)
+		if !releaseCR.IsReleased() {
+			return fmt.Errorf("release %s/%s is not marked as finished yet", releaseCR.GetNamespace(), releaseCR.GetName())
+		}
+		return nil
+	}, releaseCreationTimeout, defaultInterval).Should(Succeed())
+}
+
+func CreateComponentByCDQ(devFw framework.Framework, devNamespace, managedNamespace, fbcAppName, fbcCompName string) *appservice.Component {
+	// using cdq since git ref is not known
+	var componentDetected appservice.ComponentDetectionDescription
+	cdq, err := devFw.AsKubeAdmin.HasController.CreateComponentDetectionQuery(fbcCompName, devNamespace, fbcSourceGitUrl, "", "", "", false)
+	Expect(err).NotTo(HaveOccurred())
+	Expect(cdq.Status.ComponentDetected).To(HaveLen(1), "Expected length of the detected Components was not 1")
+
+	for _, compDetected := range cdq.Status.ComponentDetected {
+		componentDetected = compDetected
+	}
+
+	component, err := devFw.AsKubeAdmin.HasController.CreateComponent(componentDetected.ComponentStub, devNamespace, "", "", fbcAppName, false, map[string]string{})
+	Expect(err).NotTo(HaveOccurred())
+	GinkgoWriter.Println("component : ", component.Name)
+	Expect(err).ShouldNot(HaveOccurred())
+	return component
+}
+
+func CreateFBCEnterpriseContractPolicy(fbcECPName string,managedFw framework.Framework, devNamespace, managedNamespace string) {
+	defaultEcPolicySpec := ecp.EnterpriseContractPolicySpec{
+		Description: "Red Hat's enterprise requirements",
+		PublicKey:   "k8s://openshift-pipelines/public-key",
+		Sources: []ecp.Source{{
+			Name:   "Default",
+			Policy: []string{ecPolicyLibPath, ecPolicyReleasePath},
+			Data:   []string{ecPolicyDataBundle, ecPolicyDataPath},
+		}},
+		Configuration: &ecp.EnterpriseContractPolicyConfiguration{
+			Exclude:     []string{"cve", "step_image_registries", "tasks.required_tasks_found:prefetch-dependencies"},
+			Include:     []string{"minimal"},
+		},
+	}
+
+	_, err := managedFw.AsKubeDeveloper.TektonController.CreateEnterpriseContractPolicy(fbcECPName, managedNamespace, defaultEcPolicySpec)
+	Expect(err).NotTo(HaveOccurred())
+
+}
+
+func CreateFBCReleasePlanAdmission(fbcRPAName string, managedFw framework.Framework, devNamespace, managedNamespace, fbcAppName, fbcECPName, pathInRepoValue, hotfix, issueId string) {
+	var err error
+	data, err := json.Marshal(map[string]interface{}{
+		"fbc": map[string]interface{}{
+			"fromIndex":            constants.FromIndex,
+			"targetIndex":          constants.TargetIndex,
+			"binaryImage":          constants.BinaryImage,
```

Comment:
I am wondering why these 3 constants above are in a different files and all other FBC related constants are here. Any reason they are not here as well?

Reply:
I think the reason is these 3 constants may change according to the environments that the test is running. They can be passed from the environment variables in repo's constants file.